text/first-pop.md

```diff
@@ -197,12 +197,12 @@ We should probably step back and think about what we're trying to do. We want
 to:
 
 * Check if the list is empty.
-* If it's empty, just return None
-* It it's *not* empty
-    * remove the head of the list
-    * remove its `elem`
-    * replace the lists head with its `next`
-    * return `Some(elem)`
+* If it's empty, just return None.
+* It it's *not* empty,
+    * remove the head of the list,
+    * remove its `elem`,
+    * replace the lists head with its `next`,
+    * return `Some(elem)`.
```

Comment:
I'd rather not have the punctuation here if that's ok with you. I usually only punctuate longer-winded lists.